blackjack.py

```diff
@@ -64,36 +19,26 @@ def check(self):
             
     def hit(self):
         while self.count <= 13:
-            self.cards.append(self.drawcards())
+            self.cards.append(game.drawcards())
             if isinstance(self.cards[-1][0], list) == True:
                 self.count += self.cards[-1][0][1]
             else:
                 self.count += self.cards[-1][0]
 
-    def setcount(self):
-        if isinstance(self.cards[0][0], list) == True and isinstance(self.cards[1][0], list) == True:
-            self.count = self.cards[0][0][1] + self.cards[1][0][1]
-        elif isinstance(self.cards[0][0], list) == True and isinstance(self.cards[1][0], list) == False:
-            self.count = self.cards[0][0][1] + self.cards[1][0]
-        elif isinstance(self.cards[0][0], list) == False and isinstance(self.cards[1][0], list) == True:
-            self.count = self.cards[0][0] + self.cards[1][0][1]
-        else:
-            self.count = self.cards[0][0] + self.cards[1][0]
-
-class Player(GameManager):
+class Player:
      
     def __init__(self):
-        self.cards = [self.drawcards(), self.drawcards()]
+        self.cards = []
         self.bank = 1000
-        self.setcount() 
+        self.count = [] 
 
     def check(self):
         if self.count[0] > 21:
             print('\nYour hand: ', self.cards)
             print('You went over 21 and lost!')
             self.bank -= self.bid
             self.bankrupt()
-            self.restart()
+            game.restart()

```

Comment:
don't access a dependency directly like this. Pass in via the constructor (`__init__`) then use it like `self.game`